### PR TITLE
[postgres] guide improvements.

### PR DESCRIPTION
**Pull request by thigg**
Mostly rewording.
A small add for unix domain sockets.

### CODE REVIEW

File: source/guide_postgresql.rst [hunk 1]
```diff
@@ -251,6 +248,17 @@ Edit ``~/opt/postgresql/data/postgresql.conf`` and set the key values ``listen_a
                                         # (change requires restart)
 
 
+Because postgres defines the socket as abstract, you cannot see it directly in the filesystem.
```
Comment (luto) [inline source/guide_postgresql.rst:251]: Hm, I can see the socket just fine using `ls -la`. Which utility isn't able to see it? Or is this new in more recent pg versions?

**Comment by luto** [comment]
Thanks! I left a small note, otherwise this looks very good to me :)

File: source/guide_postgresql.rst (hunk 1 above)
Comment (thigg) [reply source/guide_postgresql.rst:251]: my mistake, didnt try -la because thats usually my default anyway. gonna change the text

**Comment by thigg** [comment]
fixed it

**Comment by luto** [comment]
Thanks!

**Pull request merged**